src/renderer/themes.css

```diff
@@ -1053,7 +1266,8 @@ it can be safely elided. This looks quite pleasant on this theme. */
   .mainDeepPurple.secRed,
   .mainDeepPurple.secDraculaRed,
   .mainDraculaPurple.secRed,
-  .mainDraculaPurple.secDraculaRed {
+  .mainDraculaPurple.secDraculaRed,
+  .mainSolarized.secSolarizedGreen {
```

Comment:
Add one entry to the block above to include `.mainSolarizedRed:has(.app), .secSolarizedRed:has(.app)`. Update this to be `.mainSolarizedMagenta.sec{redType}` for all red types, and same for each main red type to each `secSolarizedMagenta`, and you'll have to add the new red type of solarized red as well. We'll need to figure out an alternative for this as this is ballooning, though!

Reply:
You can do this programmatically using [a substring attribute selector](https://stackoverflow.com/a/28309152)r and changing the name of magenta to purple. If that's too complicated for now, you can just do it the manual way as well

Reply:
Update: I added this in #5071, so you don't have to worry about this!